src/SilkeyJwtPayloadVerificator.js

```diff
@@ -0,0 +1,20 @@
+import ethersjs from "ethers";
+import jwt from "jsonwebtoken";
+
+const {ethers} = ethersjs;
+
+/**
+ *
+ * @returns {null|payload}
+ * @param token
+ */
+export function silkeyJwtPayloadVerificator(token) {
+  try {
+    const payload = jwt.decode(token)
+    const signer = ethers.utils.verifyMessage(payload.email, payload.signature)
```

Comment:
since this function returns the whole payload, the whole payload should be signed, not just email

Reply:
Next release